### PR TITLE
Do not strip slash from additionalPath

### PR DESCRIPTION
**Pull request by colin-msphere**

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/prometheus/config/PrometheusConfiguration.java [hunk 1]
```diff
@@ -70,15 +70,19 @@ public boolean configure(StaplerRequest req, JSONObject json) throws FormExcepti
     }
 
     public String getPath() {
-        return StringUtils.isEmpty(additionalPath) ? urlName : urlName + "/" + additionalPath;
+        if (StringUtils.isEmpty(additionalPath)) {
+            return urlName;
+        }
+
+        return urlName + (additionalPath.startsWith("/") ? "" :  "/") + additionalPath;
```
Comment (benclarkwood) [inline src/main/java/org/jenkinsci/plugins/prometheus/config/PrometheusConfiguration.java]: Is this necessary? Won't additionalPath start with a `/`? Or has `additionalPath` been persisted somewhere and we're fixing already created ones?

File: src/main/java/org/jenkinsci/plugins/prometheus/config/PrometheusConfiguration.java [hunk 2]
```diff
@@ -70,15 +70,19 @@ public boolean configure(StaplerRequest req, JSONObject json) throws FormExcepti
...
     }
 
     public void setPath(String path) {
         // remove leading slash if it exists.
         path = path.startsWith("/") ? path.substring(1) : path;
         List<String> pathParts = Arrays.asList(path.split("/", 2));
         urlName = pathParts.get(0);
-        additionalPath = (pathParts.size() > 1) ? pathParts.get(1) : "";
+        additionalPath = (pathParts.size() > 1) ? "/" + pathParts.get(1) : "";
```
Comment (benclarkwood) [inline src/main/java/org/jenkinsci/plugins/prometheus/config/PrometheusConfiguration.java]: Not totally obvious why this is happening so maybe worth a comment.

File: src/main/java/org/jenkinsci/plugins/prometheus/config/PrometheusConfiguration.java (hunk 1 above)
Comment (colin-msphere) [reply src/main/java/org/jenkinsci/plugins/prometheus/config/PrometheusConfiguration.java]: Good point.

**Comment by benclarkwood** [comment]
@colin-msphere can we change DEFAULT_ENDPOINT to `v1/metrics/prometheus` so our testing is simpler?

**Pull request merged**